setup.py

```diff
@@ -42,17 +58,15 @@
     long_description=long_description,
     long_description_content_type="text/markdown",
     url="https://github.com/dsavransky/keplertools",
-    packages=setuptools.find_packages(exclude=['tests*']),
-    install_requires=['numpy'],
-    extras_require={
-        'C': ['cython'],
-    },
+    packages=setuptools.find_packages(exclude=["tests*"]),
+    install_requires=["numpy", "cython"],
+    extras_require={"C": ["cython"]},
```

Comment:
I'm fine with making cython an absolute requirement.  However, this makes the 'C' extra redundant.  Can you please remove it altogether?  Thanks.

Reply:
Makes sense, just pushed that change